deploy/oracles.ts

```diff
@@ -1,12 +1,26 @@
-// npx hardhat deploy --network bsctestnet
-import networks from "@venusprotocol/venus-protocol/networks/mainnet.json";
+import mainnetDeployments from "@venusprotocol/venus-protocol/networks/mainnet.json";
+import testnetDeployments from "@venusprotocol/venus-protocol/networks/testnet.json";
 import { ethers } from "hardhat";
 import { DeployFunction } from "hardhat-deploy/dist/types";
 import { HardhatRuntimeEnvironment } from "hardhat/types";
 
+const ADDRESSES = {
+  bsctestnet: {
+    vBNBAddress: testnetDeployments.Contracts.vBNB,
+    pythOracleAddress: "0xd7308b14BF4008e7C7196eC35610B1427C5702EA",
+    binanceFeedRegistryAddress: "0x999DD49FeFdC043fDAC4FE12Bb1e4bb31cB4c47B",
+  },
+  bscmainnet: {
+    vBNBAddress: mainnetDeployments.Contracts.vBNB,
+    pythOracleAddress: "",
+    binanceFeedRegistryAddress: "",
+  },
+};
+
 const func: DeployFunction = async function ({ getNamedAccounts, deployments, network }: HardhatRuntimeEnvironment) {
   const { deploy } = deployments;
   const { deployer } = await getNamedAccounts();
+  const networkName = network.name as "bsctestnet" | "bscmainnet";
```

Comment:
could you elaborate on the effects of this line? are we setting `networkName` as based on `--network` parameter from hh?

Reply:
that is correct, I've updated it so that it fallsback to testnet addresses if the network is anything but mainnet